src/Core/Houston.vala

```diff
@@ -17,85 +17,154 @@
 */
 
 public class AppCenterCore.Houston : Object {
+    public signal void refreshed ();
+
     private const string HOUSTON_API_URL = "https://developer.elementary.io/api";
+    private const string NEWEST_CATEGORY = "Newest";
+    private const string UPDATED_CATEGORY = "Updated";
+    private const string APPS_KEY = "apps";
 
     private Soup.Session session;
+    private GLib.KeyFile cache_file = null;
```

Comment:
Yeah you're right with the nullable, I don't think that it would change anything to set it as static when the Houston class is already a singleton